text/0000-lifecycle-all.md

```diff
@@ -0,0 +1,110 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle All Binary
+- Start Date: 2020-01-21
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+### `/cnb/lifecycle/all`
```

Comment:
Following suggestion in WG I am going to use this comment thread to conduct an informal poll to select the name for this binary. After this comment I will add comments each containing a single suggestion:

1. Please vote with a 👍
2. No more than 2 votes per person
3. Feel free to express dislike with a 👎 (full disclosure these may not factor into the counting)
4. To add a suggestion, add a comment containing only the suggestion

Reply:
`lifecycle/all`

Reply:
`lifecycle/make`

Reply:
`lifecycle/combo`

Reply:
`lifecycle/phases`

Reply:
`lifecycle/constructor`

Reply:
`lifecycle/creator`

Reply:
`/lifecycle/build`